scripts/set_all_parameters.R

```diff
@@ -26,12 +27,14 @@ locs <- dplyr::right_join(cw, ids, by = c("country" = "region")) ## region & cou
 locs <- dplyr::filter(locs, !is.na(region))
 countries <- unique(locs$country)
 # countries simulated in the VIMC core project
-countries <-c("AGO", "BDI", "BEN", "BFA", "CAF", "CIV", "CMR", "COD", "COG", "DZA", "ETH", "GHA", "GIN", "GNB", "KEN", "LBR", "MDG", "MLI",
-              "MOZ", "MRT", "MWI", "NAM", "NER", "NGA", "RWA", "SEN", "SLE", "SOM", "SSD", "TCD", "TGO", "TZA", "UGA", "ZAF", "ZMB", "ZWE",
-              "AFG", "HTI", "IRN", "IRQ", "NPL", "PAK", "PHL", "THA", "YEM", "IND", "BGD") #will likely to only include the countries in sub-Saharan Africa
+#countries <-c("AGO", "BDI", "BEN", "BFA", "CAF", "CIV", "CMR", "COD", "COG", "DZA", "ETH", "GHA", "GIN", "GNB", "KEN", "LBR", "MDG", "MLI",
```

Comment:
This seems possibly prone to error as we need to remember to comment / uncomment something. Is there another config setting instead that would make sense to automatically toggle between the countries list? Or perhaps we should separate out set_all_parameters for vimc core / surveillance project / custom runs? Maybe we can talk this through at our next meeting